spring-cloud-stream-modules-docs/src/main/asciidoc/processors.adoc

```diff
@@ -91,3 +91,35 @@ This transform will convert all message payloads to upper case. If sending the w
     dataflow:> http post --target http://localhost:9003 --data "foo"
 
 As part of the SpEL expression you can make use of the pre-registered JSON Path function.  The syntax is #jsonPath(payload,'<json path expression>')
+
+[[spring-cloud-stream-modules-splitter]]
+=== Splitter
+The splitter module builds upon the concept of the same name in Spring Integration and allows the splitting of a single
+message into several distinct messages.
+
+$$expression$$:: $$a SpEL expression which would typically evaluate to an array or collection$$ *($$String$$, default: `null`)*
+$$delimiters$$:: $$A list of delimiters to tokenize a String payload ('expression' must be null)$$ *($$String$$, default: `null`)*
+$$fileMarkers$$:: $$Split File payloads, when true, START and END marker messages will be emitted, when false no markers are emitted$$ *($$String$$, default: `null`)*
+$$charset$$:: $$Split File payloads using this charset to convert bytes to String$$ *($$String$$, default: `null`)*
+$$applySequence$$:: $$Add correlation and sequence information to the message headers$$ *($$String$$, default: `true`)*
+
+When no `expression`, `fileMarkers`, or `charset` is provided, a `DefaultMessageSplitter` is configured with (optional) `delimiters`.
+When `fileMarkers` or `charset` is provided, a `FileSplitter` is configured.
+Otherwise, an `ExpressionEvaluatingMessageSplitter` is configured.
+
+When splitting `File` payloads, the `sequenceSize` header is zero because the size cannot be determined at the beginning.
+
+As part of the SpEL expression you can make use of the pre-registered JSON Path function. The syntax is
+``#jsonPath(payload, '<json path expression>')``.
+
+Ambiguous properties are not allowed.
+
+==== Extract the value of a specific field
+
+This splitter converts a JSON message payload to the value of a specific JSON field.
+
+    xd:> stream create --name jsontransformtest --definition "http --port=9005 | splitter --expression=#jsonPath(payload,'$.firstName') | log" --deploy
+
+Try sending this payload to the HTTP endpoint and you should see just the value "John" in the XD log:
```

Comment:
The `XD` word is wrong here.
From other side this sample doesn't seem appropriate for the `splitter` case. Looks like it is about `transform`.
Yes, you'll see the `John` as a result, but we should have here more complicated JSON and a bit different jsonPath to demonstrate splitter. See, `JsonPathTests` in SI Core for example.